packages/dialog/test/dialog-wrapper.test.ts

```diff
@@ -127,6 +131,43 @@ describe('Dialog Wrapper', () => {
         expect(el.open).to.be.false;
         expect(closeSpy.callCount).to.equal(1);
     });
+    it("shows header divider when there's a header", async () => {
+        const wrapper = await styledFixture<DialogWrapper>(
+            wrapperWithHeadline()
+        );
+        await elementUpdated(wrapper);
+
+        const dialog = wrapper.shadowRoot.querySelector('sp-dialog') as Dialog;
+        const divider = dialog.shadowRoot.querySelector(
+            'sp-divider.divider'
+        ) as Divider;
+
+        expect(divider).to.be.not.null;
+    });
+    it('hides header divider when there\'s a header but "no-divider"', async () => {
+        const wrapper = await styledFixture<DialogWrapper>(
+            wrapperWithHeadlineNoDivider()
+        );
+        await elementUpdated(wrapper);
+
+        const dialog = wrapper.shadowRoot.querySelector('sp-dialog') as Dialog;
+        const divider = dialog.shadowRoot.querySelector(
+            'sp-divider.divider'
+        ) as Divider;
+
+        expect(divider).to.be.null;
+    });
+    it("hides header divider when there's no header", async () => {
+        const wrapper = await styledFixture<DialogWrapper>(wrapperNoHeadline());
```

Comment:
Can you add a test of this story with `await expect(el).to.be.accessible();` as well? Want to confirm there are no accessibility issues with a no-headline delivery that we'd need to handle.

Reply:
your intuition was correct!  adding the test results in the below error.  any guidance on how to fix?

```
 ❌ Dialog Wrapper > hides header divider when there's no header
      Error: Accessibility Violations
      ---
      Rule: aria-dialog-name
      Impact: serious
      ARIA dialog and alertdialog nodes should have an accessible name (https://dequeuniversity.com/rules/axe/4.4/aria-dialog-name?application=axeAPI)
      
      Issue target: sp-dialog-wrapper,sp-dialog
      Context: <sp-dialog no-divider="" dir="ltr" tabindex="0" role="dialog" aria-describedby="sp-dialog-description-55-17">
      Fix any of the following:
        aria-label attribute does not exist or is empty
        aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
        Element has no title attribute
      ---
        aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
        Element has no title attribute
        ---
        at processResults (node_modules/chai-a11y-axe/src/accessible.js:87:15)
        at node_modules/chai-a11y-axe/src/accessible.js:117:7
        at async o.<anonymous> (packages/dialog/test/dialog-wrapper.test.ts:166:8)
```

Reply:
@jnurthen if we have a dialog without a visible headline, can we promote the body content to the `aria-labelledby` content to appropriately fulfil the accessibility contract here? Visually, imagine a prompt or alert.

Reply:
@beeduul FYI the Spectrum spec lists the [title as "required"](https://spectrum.adobe.com/page/alert-dialog/#Title) but I'll reach out to the design team as that feels somewhat constrictive. In the meantime, hopefully we'll hear from the accessibility team on the feasibility of a Dialog without a title in that context.

Reply:
@beeduul with this information the pattern we've namely been using would be to add a `label` attribute to the `<sp-dialog-wrapper>` and `<sp-dialog>` elements. The Wrapper would namely pass it down to the Dialog, and the Dialog would apply it to itself as an `aria-label` attribute. This would be a good place to install a Dev Mode warning to support consumers in accurately delivering this pattern to their visitors, similar to what can be found at https://github.com/adobe/spectrum-web-components/blob/main/packages/progress-circle/src/ProgressCircle.ts#L122-L142